bundle.yaml

```diff
@@ -1,58 +1,58 @@
 series: trusty
 services:
   etcd:
-    charm: "cs:~containers/trusty/etcd-2"
+    charm: cs:~containers/trusty/etcd-2
     num_units: 3
     annotations:
-      "gui-x": "1200"
-      "gui-y": "600"
-    constraints: "arch=amd64 cpu-cores=2 mem=4G root-disk=12G"
+      gui-x: '1200'
+      gui-y: '600'
+    constraints: arch=amd64 cpu-cores=2 mem=4G root-disk=12G
```

Comment:
I'm not sure about the formatting changes in this file, but if it still works, then it's fine.